src/app/(auth)/(routes)/sign-in/[[...sign-in]]/components/user-auth-form/use-user-auth-form.ts

```diff
@@ -79,6 +79,7 @@ export const useUserAuthForm = ({ redirectUrl }: UseUserAuthFormProps) => {
       if (error.message === 'Session already exists') {
         router.push(redirectUrl || '/dashboard')
       }
+      console.log('Error authenticating')
```

Comment:
Consider using `console.error` and passing the `error` object as an argument (when it's applicable). This approach would provide more detailed information about the root cause of the issue, enhancing debugging clarity.

Example: 

```typescript
console.error('An error occurred:', error);
```